planner/core/encode.go

```diff
@@ -64,5 +71,95 @@ func (pn *planEncoder) encodePlan(p PhysicalPlan, isRoot bool, depth int) {
 	case *PhysicalIndexLookUpReader:
 		pn.encodePlan(copPlan.indexPlan, false, depth)
 		pn.encodePlan(copPlan.tablePlan, false, depth)
+	case *PhysicalIndexMergeReader:
+		for i := 0; i < len(copPlan.partialPlans); i++ {
```

Comment:
How about using range?
for _, partialPlan := range copPlan.partialPlans {
pn.encodePlan(partialPlan, false, depth)
}

Reply:
done.